docs/types/Store.prototype.createData.md

```diff
@@ -3,8 +3,38 @@
 
 @signature `Store.prototype.createData(request, response)`
 
-A `requestHandler` that creates an item in the store.
+  A `requestHandler` that creates an item in the store.
```

Comment:
This should link to requestHandler.

Also, I'd say that `createData` creates records in the store.  It can serve as a `requestHandler`.

Reply:
Resolved in 162c8fa